designer/report-designer/src/main/resources/org/pentaho/reporting/designer/core/actions/messages/messages.properties

```diff
@@ -644,7 +644,7 @@ PreviewExcelAction.Text=Excel
 PreviewXlsxAction.Accelerator=
 PreviewXlsxAction.Mnemonic=E
 PreviewXlsxAction.Description=Preview the Report as a Excel 2007 file
```

Comment:
We can remove the '2007' from the description too - and, at the same time, please correct "a Excel" to "an Excel" :)